soh/soh/resource/importer/AudioSampleFactory.cpp

```diff
@@ -66,6 +73,62 @@ void LUS::AudioSampleFactoryV0::ParseFileBinary(std::shared_ptr<BinaryReader> re
     audioSample->book.book = audioSample->bookData.data();
     audioSample->sample.book = &audioSample->book;
 }
+
+void LUS::AudioSampleFactoryV1::ParseFileBinary(std::shared_ptr<BinaryReader> reader,
+                                                 std::shared_ptr<IResource> resource)
+{
+    std::shared_ptr<AudioSample> audioSample = std::static_pointer_cast<AudioSample>(resource);
+    ResourceVersionFactory::ParseFileBinary(reader, audioSample);
+
+    reader->Seek(52, SeekOffsetType::Start);
+    if (reader->ReadChar() == 'W' &&
+        reader->ReadChar() == 'A' &&
+        reader->ReadChar() == 'V' &&
+        reader->ReadChar() == 'E') {
+        reader->Seek(-12, SeekOffsetType::Current);
+        u32 curr = reader->GetBaseAddress();
+        reader->Seek(0, SeekOffsetType::End);
+        u32 end = reader->GetBaseAddress();
+        reader->Seek(curr, SeekOffsetType::Start);
+        u32 len = end - curr + 1;
+        u8* strem2 = (u8 *) malloc(sizeof(u8) * len);
+        reader->Read((char *) strem2, len);
+        drwav_uint32 channels;
+        drwav_uint32 sampleRate;
+        drwav_uint64 totalPcm;
+        drwav_int16* pcmData =
+            drwav_open_memory_and_read_pcm_frames_s16(strem2, len, &channels, &sampleRate, &totalPcm, NULL);
+        audioSample->sample.size = totalPcm;
+        audioSample->sample.sampleAddr = (uint8_t*)pcmData;
+        audioSample->sample.codec = 5; //CODEC_S16
```

Comment:
Would be better to include the z64audio.h if its not accesible and use the enums rather than numbers even if commented. I also think explicitly setting sample.medium to MEDIUM_RAM seems like the best practice here? (unless there's some reason you would want it not to be)